src/main/java/seedu/address/logic/commands/HelpCommand.java

```diff
@@ -6,15 +6,22 @@
  * Format full help instructions for every command for display.
  */
 public class HelpCommand extends Command {
-    public static final String ADD_COMMAND = "add c/CS1231 cr/4 t/Core : Adds module 'CS1231'"
-            + "with 4 credits and a 'Core' tag to it.";
+    public static final String ADD_COMMAND = "add CS1231 t/Core : Adds module 'CS1231' with 'Core' tag to it.";
     public static final String HELP_COMMAND = "help : Displays the help guide.";
-    public static final String LIST_COMMAND = "list : Displays list of current modules stored";
-    public static final String EDIT_COMMAND = "edit 1 c/CS1231 cr/4 :"
-            + "Replaces the current module at index 1 with module 'CS1231' with 4 credits.";
-    public static final String FIND_COMMAND = "find CS1231 : Searches for the 'CS1231' module.";
-    public static final String DELETE_COMMAND = "delete 1 : Deletes the module at index 1.";
+    public static final String LIST_COMMAND = "list : Displays list of current modules stored.";
+    public static final String EDIT_COMMAND = "edit CS1231 c/CS1101S :"
+            + "Replaces the module 'CS1231' with module 'CS1101S'.";
+    public static final String FIND_COMMAND = "find CS1 : Displays all modules that contain the prefix 'CS1'. \n"
+            + "find core : Displays all modules that contain the 'core' tag.";
+    public static final String DELETE_COMMAND = "delete CS1231 : Deletes the 'CS1231' module from your GradPad.";
     public static final String CHECKMC_COMMAND = "checkmc : Checks the current total credits achieved by all modules.";
+    public static final String REQUIRED_COMMAND = "required : Displays left over required modules that you have yet"
+            + " to clear.";
+    public static final String GEM_COMMAND = "gem : Displays all available General Education modules by the Semester.";
+    public static final String SCIENCE_COMMAND = "science : Displays all available Science modules.";
+    public static final String SEARCH_COMMAND = "search MA1521 : Searches and displays all relevant information about"
```

Comment:
Should we boast and say "from NUSMods" ah LOL

Reply:
MIGHT AS WELL TBH